### PR TITLE
Add pagination for results > 10

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
**Issue:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/4149
**Issue 2:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/4378

**Current Behavior:** No pagination when there's more than 10 search results on http://localhost:3001/find-va-forms-beta.

**Desired Behavior:** Pagination when there's more than 10 search results on http://localhost:3001/find-va-forms-beta.

This PR implements the above behavior. This PR also implements the new columns from [this issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/4378) now.

## Testing done
Locally.

#### Displaying {start} - {end} out of {total} results for {searchTerm}
image

#### Displaying last page of results
image

## Acceptance criteria
- [x] Add Formation "Pagination" component for the above page (You can find this [here](https://design.va.gov/components/pagination).)
- [x] Add "Displaying x - y out of z results for {query}" above the table
- [x] "Issue Date" and "Revision Date" columns now replace "Description" and "Available Online" columns.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/find-va-forms/containers/SearchResults.jsx [hunk 1]
```diff
@@ -79,11 +84,32 @@ class SearchResults extends Component {
...
 
     // Update local state for SortableTable.
-    this.setState({ selectedFieldLabel: fieldLabel, selectedFieldOrder });
+    this.setState({
+      selectedFieldLabel: fieldLabel,
+      selectedFieldOrder,
+    });
+
+    // Reset pagination values.
+    updatePagination();
```
Comment (ncksllvn) [inline src/applications/find-va-forms/containers/SearchResults.jsx]: Can we add a unit test for this function, and for the `Pagination` component during render?

**Review by ncksllvn — APPROVED** [review]
LGTM!

**Review by rianfowler — APPROVED** [review]

**Comment by kelsonic** [comment]
Jenkins is failing but it doesn't look like it's related to my code...
image

**Comment by kelsonic** [comment]
This PR now includes https://github.com/department-of-veterans-affairs/va.gov-team/issues/4378.

**Pull request merged**